Formula/gocryptfs.rb

```diff
@@ -0,0 +1,29 @@
+class Gocryptfs < Formula
+  desc "Encrypted overlay filesystem written in Go"
+  homepage "https://nuetzlich.net/gocryptfs/"
+  url "https://github.com/rfjakob/gocryptfs/releases/download/v1.4.3/gocryptfs_v1.4.3_src-deps.tar.gz"
+  version "1.4.3"
+  sha256 "adf55824fd6ca8e258e9207217c52206953293c6ff86e68b58874b6d195e5b11"
+
+  depends_on "go" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl"
```

Comment:
what is the benefit of using this and not `build-without-openssl.bash`?

Reply:
It is a lot faster (4x).